docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -1 +1,98 @@
+
+**commit** +
+A change to one or more files. In {ProductName}, commits that you make to a linked git repository move through the pipeline and automatically get published.
+
+**component** +
+An image that {ProductName} builds from source code in a repository. One or more components that run together form an application.
+
+**Conftest** +
+A utility for testing structured configuration data. Use Conftest to validate container information.
+
+**enterprise contract (EC)** +
+A set of release policies that you apply to your application snapshot. Use the EC to prevent releases that are not compliant with Supply-chain Levels for Software Artifacts (SLSA) guidelines. 
+
+
+**integration test** +
+A pipeline that you set up in GitHub. When you add components, it tests each one individually, and then tests the application as a whole.
+
+**IntegrationTestScenario** +
+A parameter that the integration service uses to test application components.
+
+**Java build service** +
+A group of utilities that accelerates build times, locates dependencies in final images, and meets product security requirements by managing the construction of Java dependencies. 
+
+**lifecycle run** +
+The processes that automatically occur after a merge or pull request is resolved. These processes can include build tests, integration tests, and TaskRuns. 
+
+**managed workspace** +
+A workspace that mirrors some details of an already existing workspace. Supporting teams can create a managed workspace to grant limited permissions to development teams in the production environment. 
+
+**OSBS features** +
+The layered image build service, or OSBS (derived from OpenShift Build Service), is a tool for building layered images. With OSBS features, you can build base RHEL images, layered images, and the builder image itself.
+
+**persister** +
+A component that moves all of the relevant PipelineRun information, known as the Pipeline output, to an external storage that is outside of the cluster’s etcd database. The persister runs after the system completes the PipelineRun.
+
+**pipelines as code** +
+A practice that defines pipelines by using source code in Git. Pipelines as Code is also the name of link:https://pipelinesascode.com[a subsystem] that executes those pipelines.
+
+**pipeline results** +
+Systems that retain the history and details of builds. 
+
+**PipelineRun** +
+A collection of TaskRuns that are arranged in a specific order of execution. A valid pipeline is compliant with the customer’s enterprise contract.
+
+**pipeline service** +
+A managed service that securely runs Tekton pipelines. It also stores the metadata and pipeline logs that are related to an executed pipeline in a separate database.
+
+**product security test** +
+A test that assesses the security of build inputs and outputs with vulnerability scanning, code scanning, image testing, and result reporting. It also identifies outdated images, applications, and build pipelines. 
```

Comment:
Calling this a "product" security test seems to bleed out Red Hatisms. Should we call this out specifically as a different type of build-time test? Or just leave them as security-focused build-time tests?

Reply:
I'm gonna remove this definition for now, because I think the new doc about build-time tests does a good job of introducing and defining both build-time tests in general, and highlighting the security tests under that category.